### PR TITLE
fix: jarring effect in Tooltip component

### PR DESCRIPTION
**Pull request by karthik-selva**
## Description
Tooltip uses the Dropdown component for dynamic positioning. When tooltip icon is hovered, the tooltip content is positioned overlapping on the icon, which creates a jarring effect.

## Motivation and Context
Users can easily see the tooltip text without the jarring effect.

## How Has This Been Tested?
Tested the change in the browser since its a CSS fix

## Checklist:
- [ ] I have added a tag to this pull request that indicates the impact of the change (patch, minor or major)
- [x] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly. (This includes updating the changelog).
- [ ] I have read the **CONTRIBUTING** document. (Last updated 24 Mar 2020)
- [ ] All my changes are covered by tests.
- [ ] This request is ready to review and merge

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -55,8 +55,8 @@ running this script and should not be pushed to the repository.
 Once you see a line like `[TIMESTAMP] waiting for changes...`, hexagon has
 finished building and you can navigate to the following URLs in your browser:
 
-- `http://localhost:10011/demo/` - The built version of `src/demo`
-- `http://localhost:10011/test/` - The tests, built from `src/hexagon.spec.js`
+- `http://localhost:10001/demo/` - The built version of `src/demo`
+- `http://localhost:10001/test/` - The tests, built from `src/hexagon.spec.js`
```
Comment (c-frater) [inline .github/CONTRIBUTING.md]: 👍 Thanks - Didn't even notice this was wrong

File: src/components/tooltip/theme.css [hunk 2]
```diff
@@ -14,6 +14,10 @@
   margin-top: var(--tooltip-space);
 }
 
+.hx-dropdown.hx-dropdown-up{
+    margin-top: calc(-1 * var(--tooltip-space));
```
Comment (c-frater) [inline src/components/tooltip/theme.css]: Could you also remove the `margin-bottom` rule(s) form the `.hx-tooltip` and `.hx-tooltip.hx-dropdown-down` as they aren't needed with this change and can cause more overlapping issues 😓

Comment (karthik-selva) [reply src/components/tooltip/theme.css]: Yes, I checked out and it appears they're good to get rid off.  Adding the change

**Comment by c-frater** [comment]
I'll add docs when I release later today

**Pull request merged**